db/migrate/20250123151657_add_path_to_trips.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPathToTrips < ActiveRecord::Migration[8.0]
+  def change
+    add_column :trips, :path, :line_string, srid: 3857
```

Comment:
I don't think 3857 is the correct value to use. I think it should be 4326.
Ref: https://gis.stackexchange.com/a/380296/55872

Not sure it matters right now (i.e. it probably works with 3857 too for now), but it might cause issues later, if you try to use PostGIS functions like `ST_Contains()`.

Reply:
I'm not an expert in GIS, but I did my small research and came to a conclusion that 3857 serves well in case of flat planar projections, specifically for web mapping. But as I said, not an expert, so if you could provide pro/contra for 4326, I may consider changing it in the future. Thank you!